src/charm.py

```diff
@@ -94,6 +94,34 @@
 
 PYDANTIC_IS_V1 = int(pydantic.version.VERSION.split(".")[0]) < 2
 
+# Regex for Kubernetes annotation values:
```

Comment:
Do you have links where these come from?  If yes, that's a good thing to add in the comments